src/importers/azureDevops.js

```diff
@@ -63,9 +69,27 @@ export async function getOrganizationInfo() {
   return result;
 }
 
-export async function getWorkItems(organization, offset) {
+export async function getProjectInfo(organization) {
+  if (!organization) {
+    return []
+  }
+
+  let endPoint = `https://dev.azure.com/${organization}/_apis/projects?api-version=6.0`;
+  let forWhat = "projectInfo";
+  const result = await sendGetHttpRequest(endPoint, forWhat);
+
+  return result;
+}
+
+export async function getWorkItems(organization, project, offset) {
   const body = {
-    query: "Select  [System.Id], [System.Title], [System.State] From WorkItems",
+    query: `
+      Select  [System.Id], [System.Title], [System.State]
+      From WorkItems
+      Where [State] <> 'Done'
```

Comment:
Is `Done` an internal value/key, or is it configurable by the user?

Reply:
It is user-configurable, yea... but I don't see a way to filter by status category with ADO's query language 😕

Reply:
Found it! Status categories aren't a separate field. They're available via a separate operator on the status field.